fluentgen/src/main/resources/Manager_Configurable.txt

```diff
@@ -9,6 +9,7 @@ public static final class Configurable {
     private final List<HttpPipelinePolicy> policies = new ArrayList<>();
     private final List<String> scopes = new ArrayList<>();
     private RetryPolicy retryPolicy;
+    private RetryOptions retryOptions;
```

Comment:
what is this .txt used for ? does it publish somewhere？

Reply:
This is the Java code in Manager. It is loaded into Template.